src/components/AdminKyc/index.tsx

```diff
@@ -50,13 +51,17 @@ const Row: FC<RowProps> = ({ item, openModal }: RowProps) => {
     createdAt,
     updatedAt,
     individualKycId,
+    audits,
   } = item
 
   const kyc = individualKycId ? item.individual : item.corporate
   const completedKycOfProvider = item?.individual?.completedKycOfProvider
   const fullName = individualKycId
     ? [kyc?.firstName, kyc?.lastName].filter((el) => Boolean(el)).join(' ')
     : kyc?.corporateName
+  const approverUser = audits.length > 0 && audits[audits.length-1]?.approvedByUser || audits[audits.length-1]?.rejectedByUser
```

Comment:
@galvin96I do you think it's better if we use ternary condition here. We should favor to use ternary condition instead of && that can sometimes give unpredictable output?
```suggestion
 const approverUser = audits.length > 0
  ? audits[audits.length - 1]?.approvedByUser || audits[audits.length - 1]?.rejectedByUser
  : null;
```

Reply:
Nope. the (&&) operator could return undefined value if the audit list is empty, it is predictable. At the next step, I also checked the approverUser variable before processing, it's safe.